### PR TITLE
Feature/company screen landscape

### PR DESCRIPTION
**Pull request by roxanap33**

### CODE REVIEW

**Review by adrianacostep33 — APPROVED** [review]
Looks good

File: MarketMatrix/src/components/chart/ChartConfiguration.tsx [hunk 1]
```diff
@@ -8,23 +8,25 @@ interface ChartConfigurationProp {
...
+
   const chartConfig = {
     chart: {
       type: chartType,
       backgroundColor: Colors.companyScreenBackground,
       style: {
-        fontFamily: 'Arial',
         color: '#ffffff',
-        fontSize: 28,
+        fontSize: height ? (height > 500 ? '28' : '14') : '28',
```
Comment (ioanagirbap33) [inline MarketMatrix/src/components/chart/ChartConfiguration.tsx:29]: Maybe you should write here just: height > 500 ? '28' : '14'

File: MarketMatrix/src/components/chart/ChartConfiguration.tsx [hunk 2]
```diff
@@ -70,7 +72,7 @@ export const ChartConfiguration = ({
           color: '#ffffff',
           style: {
             fontWeight: 'bold',
-            fontSize: 24,
+            fontSize: height ? (height > 500 ? '24' : '14') : 24,
```
Comment (ioanagirbap33) [inline MarketMatrix/src/components/chart/ChartConfiguration.tsx:75]: Here too

File: MarketMatrix/src/components/chart/ChartConfiguration.tsx [hunk 3]
```diff
@@ -86,7 +88,7 @@ export const ChartConfiguration = ({
       enabled: true,
       itemStyle: {
         color: '#ffffff',
-        fontSize: 24,
+        fontSize: height ? (height > 500 ? '24' : '14') : '24',
```
Comment (ioanagirbap33) [inline MarketMatrix/src/components/chart/ChartConfiguration.tsx:91]: Here too

**Review by ioanagirbap33 — APPROVED** [review]

**Pull request merged**